kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ProcessServiceIntegrationTest.java

```diff
@@ -247,6 +247,38 @@ public void testSignalProcessInstanceNullEvent() throws Exception {
 
     }
 
+    @Test
+    public void testBoundarySignalProcessInstance() throws Exception {
+        Long processInstanceId = processClient.startProcess(CONTAINER_ID, PROCESS_ID_BOUNDARY_SIGNAL_PROCESS);
+        assertNotNull(processInstanceId);
+        assertTrue(processInstanceId.longValue() > 0);
+
+        try {
+            checkAvailableBoundarySignals(CONTAINER_ID, processInstanceId);
+        } catch (Exception e){
+            processClient.abortProcessInstance(CONTAINER_ID, processInstanceId);
```

Comment:
Please move abortProcessInstance into the final statement, it should be done in any case.
Also please delete e.printStackTrace();  - it doesn't bring any value.
Also please delete fail(e.getMessage()); - unexpected exceptions should be thrown out of test method, JUnit will handle it as a failing test, printing exception with stacktrace to the log.